editor/filesystem_dock.cpp

```diff
@@ -3261,9 +3266,27 @@ void FileSystemDock::_file_and_folders_fill_popup(PopupMenu *p_popup, const Vect
 	if (p_paths.size() == 1) {
 		const String &fpath = p_paths[0];
 
-#if !defined(ANDROID_ENABLED) && !defined(WEB_ENABLED)
 		p_popup->add_separator();
```

Comment:
I believe this separator will now end up looking a bit odd on Android and web builds in the case where you're *not* "right-clicking" a favorite.

The permutations are basically:

1. A desktop build where it's *not* a favorite.
2. A desktop build where it *is* a favorite.
3. An Android/web build where it's *not* a favorite.
4. An Android/web build where it *is* a favorite.

You most likely want a separator only for cases 1, 2 and 4, whereas the PR currently adds a separator to all four of them.

Reply:
done